developer-tools/ReducedLocalStack/src/main/kotlin/org/dataland/reducedstack/ReducedStack.kt

```diff
@@ -0,0 +1,99 @@
+package org.dataland.reducedstack
+
+import freemarker.template.Configuration
+import java.io.File
+import java.io.FileWriter
+import java.io.Writer
+
+private val selectedServices =
+    mapOf(
+        "proxy" to true, // always required
+        "adminProxy" to true, // always required
+        "apiKeyManager" to false,
+        "documentManager" to true,
+        "backendDb" to true, // always required
+        "internalStorage" to true, // always required
+        "keycloak" to true, // always required
+        "pgadmin" to false,
+        "rabbitmq" to true, // always required
+        "qaService" to true,
+        "userService" to false,
+        "batchManager" to false,
+        "communityManager" to true,
+        "specificationService" to true,
+        "emailService" to false,
+        "externalStorage" to false,
+        "eurodatClient" to false,
+        "dataExporter" to false,
+        "grafana" to false,
+        "loki" to false,
+        "alloy" to false,
+        "frontendDev" to false, // if this is set to false, use -l flag and start the frontend dev server manually
```

Comment:
```suggestion
        "frontendDev" to true, // if this is set to false, use -l flag and start the frontend dev server manually
```
(wie im Hinweis oben)

Reply:
done